custom_components/alexa_media/__init__.py

```diff
@@ -774,6 +809,7 @@ async def ws_handler(message_obj):
                         f"{DOMAIN}_{hide_email(email)}"[0:32],
                         {"push_activity": json_payload},
                     )
+                    await coord.async_request_refresh()
```

Comment:
Will there be a race condition with the dispatcher above? We used to rely on the receiving entity to do the refresh. If they're waiting for the coordinator, should the async_dispatcher be moved after this call? It's also possible we don't even need to dispatch anymore because the coordinator can provide that automatically..

Reply:
The async_dispatcher call is still necessary as it is still received by the media player entities. I tested and investigated the data race concern and I believe there isn't a problem. In general, I think async_request_refresh is safe to call all the time. it is debounced and delayed automatically.

Reply:
Can we just swap the order so the request refresh is sent before informing the media players? Dealing with the race condition later would be painful since it would happen randomly for random people.

Reply:
Order swapped.